fetch-content.sh

```diff
@@ -9,7 +9,7 @@ else
 	DIR="$HOME/.openra/Content/ra2/"
 fi
 
-if [ ! -e "$DIR" ]; then
+if [ -z "$(ls -A "$DIR")" ]; then
```

Comment:
If directory doesn't exist then it will output to stderr.  Perhaps,

```bash
if [ ! -e "$DIR" -o -z "$(ls -A "$DIR")" ]; then
...
```
or discarding stderr of `ls -A`.